projects/igniteui-angular/src/lib/grids/pivot-grid/pivot-grid.component.ts

```diff
@@ -745,7 +745,7 @@ export class IgxPivotGridComponent extends IgxGridBaseDirective implements OnIni
 
     /**
      * @hidden @internal
-     * @deprecated in version 18.1.0. Use the `autoGenerate` property instead.
```

Comment:
Heh, if it's `@hidden @internal` the deprecation message perhaps is not needed, but sure in the off chance it's discovered in some IDE API listing, don't mind it staying :)